shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/show/executor/ShowAuthorityRuleExecutor.java

```diff
@@ -55,7 +55,7 @@ protected MergedResult createMergedResult() {
         }
         AuthorityRuleConfiguration authorityRuleConfiguration = authorityRuleConfigurationOptional.get();
         List<Object> row = new LinkedList<>();
-        row.add(authorityRuleConfiguration.getUsers().stream().map(each -> each.getGrantee() + ":" + each.getPassword()).collect(Collectors.joining("; ")));
+        row.add(authorityRuleConfiguration.getUsers().stream().map(each -> each.getGrantee() + "").collect(Collectors.joining("; ")));
```

Comment:
Why is `+ ""` needed here, can it be removed or replaced by toString()?

Reply:
Of course you can, immediately change to toString()